### DISCUSSION TITLE
Multiple relations between models

### DISCUSSION THREAD

**Discussion by Kay2dan** [prisma/prisma#3889] [Q&A] [answered]
I am struggling to understand how to build multiple relations between two models. Take the following scenario:

Model 'Board' can have a user in the 'owner' field, a number of users in 'invited' field and a number of users in 'invitee' field. How would I create this in structure in schema?

```prisma
model User {
  id             Int       @default(autoincrement()) @id
 board      Board[]
  ...
}

model Board {
  id             Int           @default(autoincrement()) @id
  owner     User        @relation(fields: [ownerId], references: [id])
  ownerId  Int
  invitee     User[]      @relation(name: "invitee") //?
  invited     User[]      @relation(name: "invited") //?
  ...
  @@unique([goalOwnerId, active])
}
```

I am trying to make the invitee & invited fields to accept multiple Users.

Thank you.

**Comment by ryands17** [comment, marked as answer]
@Kay2dan 👋 

If a user can invite and be invited to multiple boards, it would look something like this:

```prisma
model User {
  id            Int     @id @default(autoincrement())
  ownedBoards   Board[]
  inviteeBoards Board[] @relation("invitee", references: [id])
  invitedBoards Board[] @relation("invited", references: [id])
}

model Board {
  id      Int    @id @default(autoincrement())
  owner   User   @relation(fields: [ownerId], references: [id])
  ownerId Int
  invitee User[] @relation(name: "invitee")
  invited User[] @relation(name: "invited")
}

```

**Reply by Kay2dan** [reply]
Thank you as always Ryan. Much appreciated!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
